config/config.exs

```diff
@@ -5,6 +5,11 @@
 # is restricted to this project.
 use Mix.Config
 
+config :sing_for_needs, SingForNeeds.Repo,
+  database: "sing_for_needs_repo",
+  username: System.get_env("POSTGRES_USERNAME") || "postgres",
+  password: System.get_env("POSTGRES_PASSWORD") || "postgres",
+  hostname: "localhost"
```

Comment:
I'm not sure you want this if you are just going to overwrite it in dev, test, and prod

Reply:
@SherSpock Good catch! We don't need it.